### ISSUE TITLE
Map continuation 

### ISSUE THREAD

**Issue by iyousifi** [SUHAFUTI/Glorious-Console-Adventure#12]
We need to make sure that we never have dead ends. 

Only solution coming to mind right now is a global unexplored exits counter, the lower the counter higher the chance of exit generated.

**Comment by uberEpicDonkey** [comment]
By dead ends you mean we want to ensure that we can always go somewhere right? We still want caves with no exits?

**Comment by iyousifi** [comment]
Yes cave with no exits are fine, as long as it's not the last explored cave. If we have unexplored caves we have potential exits, so if UnexploredMaps == 0 and we step into a new map we need that map to generate exits to ensure map/game continuation.

**Comment by uberEpicDonkey** [comment]
How do we know if there are any unexplored maps left? We just add to the counter pr. exit?

**Comment by iyousifi** [comment]
Yes I'm thinking the maps know of their exits and if they're explored adding to a global counter upon creation. Once it's explored we subtract from the global counter. Just so the knowledge of exploration is with the maps. Cause if a map is revisited it knows what exits are explored.

**Comment by iyousifi** [comment]
kinda fixed with dynamite
